crates/biome_js_analyze/src/lint/style/use_naming_convention.rs

```diff
@@ -471,21 +538,15 @@ declare_lint_rule! {
     ///                 },
     ///                 "match": "_(.+)",
     ///                 "formats": ["camelCase"]
-    ///             },
-    ///             {
-    ///                 "selector": {
-    ///                     "kind": "enumMember"
-    ///                 },
-    ///                 "formats": ["PascalCase", "CONSTANT_CASE"]
     ///             }
     ///         ]
     ///     }
     /// }
     /// ```
     ///
     /// If `match` is set and `formats` is unset,
-    /// then the part of the name captured by the regular expression is forwarded to the next conventions of the array.
-    /// In the following example, we require that private class members start with `_` and all class members are in ["camelCase"].
+    /// then the part of the name captured by the regular expression is forwarded to the next conventions of the array that selects the declaration.
+    /// The following configuration has exactly the same effect as the previous one.
```

Comment:
```suggestion
    /// then the part of the name captured by the regular expression is forwarded to the next captured conventions.
    /// The following configuration has exactly the same effect as the previous one.
```
 See if the suggestion makes sense. As for the second phrase, please explain why the effect is the same.